test/aws/rails/mailer_test.rb

```diff
@@ -36,8 +32,10 @@ module Rails
         TestMailer.deliverable(
           body: 'Hallo',
           from: 'sender@example.com',
```

Comment:
Do you think we should add a test that checks the defaults of from/to methods when not using smtp methods?